swarm/network/stream/delivery.go

```diff
@@ -271,7 +272,7 @@ func (d *Delivery) RequestFromPeers(ctx context.Context, req *network.Request) (
 		Addr:      req.Addr,
 		SkipCheck: req.SkipCheck,
 		HopCount:  req.HopCount,
-	}, Top)
+	}, Top, "request.from.peers")
```

Comment:
@nolash I didn't see this at all in Jaeger, so I am not sure this part is working as expected.